### ISSUE TITLE
Back end: visualization endpoints should return number of incomplete cases even if `showMissingness: true` is not requested by the client

### ISSUE THREAD

**Issue by bobular** [VEuPathDB/plot.data#80]
From Slack
>  I don't think I have all the information I need in a (e.g.) scatterplot response to do the Birds Eye View barplot :disappointed:
The short version is that if I have an overlay variable active, I need to calculate "number complete for x+y vars".   `config.completeCases` is not the number we need (this is the value for "number complete for x+y+strata".
I can request `showMissingness: 'TRUE'` and use `config.completeCases + config.plottedIncompleteCases` but having `showMissingness` true all the time messes up proportional histograms and barplots.  I can give more details and examples in the thread if needed

This is to get the correct info for the [Bird's Eye View](https://github.com/VEuPathDB/web-eda/issues/234) plot.

Note that the numbers should be the same when `showMissingness` is true or false (currently `plottedCompleteCases` is zero when `showMissingness` is false).

> I think something like config.incompleteCases would do the trick!

Likely for plot.data and @d-callan 
Urgency: moderate

**Comment by nkittur-uga** [comment]
This is a dependency of Bob's being able to implement the birds eye bar. Not something that can be QAed on the front end. Pausing this ticket.

**Comment by bobular** [comment]
This will be testable when https://github.com/VEuPathDB/web-eda/pull/376 is approved, merged and deployed.

**Comment by danicahelb** [comment]
@nkittur-uga this should be ready to QA now

**Comment by bobular** [comment]
Yeah, please carefully check all the numbers!  I think they are all correct
now.

**Comment by nkittur-uga** [comment]
Passed QA. All numbers are as expected both in bird's eye view and in the title "xx Repeated measures". Also tested with toggle button for "including repeated measures with no data for stratif..." and title numbers change as expected while bird's eye plot does not change.

This is SO NICE!
